Octokit.Reactive/Clients/IObservableRepositoryCommitsClients.cs

```diff
@@ -17,6 +23,16 @@ public interface IObservableRepositoryCommitsClient
         IObservable<CompareResult> Compare(string owner, string name, string @base, string head);
 
         /// <summary>
+        /// Compare two references in a repository
+        /// </summary>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="base">The reference to use as the base commit</param>
+        /// <param name="head">The reference to use as the head commit</param>
+        /// <returns></returns>
```

Comment:
I should have been clearer - the whole `<returns></returns>` element can be removed from these docs.